src/main/Interval.js

```diff
@@ -39,6 +39,16 @@ class Interval {
     return this.contains(other.start) && this.contains(other.stop);
   }
 
+  // Expand range to begin and end on multiples of size
+  expand(size: number, zeroBased: boolean): Interval {
+    var minimum = zeroBased ? 0 : 1;
+    var roundDown = x => x - x % size;
```

Comment:
can you add a simple example to what this line would do as a comment? And maybe encapsulate the modulo operation in parentheses just for clarity?

This is just me thinking loud: the `expand` name seems a bit misleading here: I first thought that this was just expanding the region by the given size, but this might actually not expand one of the ends if either the `start` or `stop` are already multiples of a size - e.g.: `(10, 20) -> expand(5) -> (10, 20)`. Maybe using `round` in the name would be better but I leave it up to you since this was what it was named originally.